src/index.js

```diff
@@ -9,9 +9,14 @@ import { Auth0Provider } from "@auth0/auth0-react";
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
   <React.StrictMode>
-    <Auth0Provider>
-      {" "}
-      <App />{" "}
+    <Auth0Provider
+      domain="indexguru.us.auth0.com"
```

Comment:
Add `domain` and `clientId` in `.env`, both are sensitive details, they should not be available on GitHub.

Reply:
I added the domain and client ID in .env